docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+Read the Docs tutorial
+======================
+
+In this tutorial you will create a documentation project on Read the Docs
+by importing an Sphinx project from a GitHub repository,
+tailor its configuration, and explore several useful features of the platform.
+
+The tutorial is aimed at people interested in learning
+how to use Read the Docs to host their documentation projects.
+You will fork a fictional software library
+similar to the one developed in the :doc:`official Sphinx tutorial <sphinx:tutorial/index>`.
+No prior experience with Sphinx is required,
+and you can follow this tutorial without having done the Sphinx one.
+
+The only things you will need to follow the are
+a web browser, an Internet connection, and a GitHub account
+(you can `register for a free account <https://github.com/signup>`_ if you don't have one).
+You will use Read the Docs Community, which means that the project will be public.
+
+Getting started
+---------------
+
+Preparing your project on GitHub
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To start, `sign in to GitHub <https://github.com/login>`_
+and navigate to `the tutorial GitHub template <https://github.com/astrojuanlu/tutorial-template/>`_,
+where you will see a green "Use this template" button.
+Click it to open a new page that will ask you for some details:
+
+* Leave the default "Owner", or optionally change it if you desire.
+* Introduce an appropriate "Repository name", for example ``rtd-tutorial``.
+* Make sure the project is "Public", rather than "Private".
+
+.. figure:: /_static/images/tutorial/github-template.png
```

Comment:
Seems weird to have this figure when the content above is talking about a different page (the "Use this template" page)

Reply:
Woops, true. Will move it down.